src/services/events/worker.js

```diff
@@ -78,11 +79,12 @@ const send = async (userId, message, body, id, identityName, setting) => {
 
   if (setting.email) {
     try {
-      await email(body.type, userId, message, id, identityName)
-    } catch(err) {
+      await email(body.type, userId, message, id, identityName, {
+        additionalKwargs: body.additionalKwargs ?? {}
+      })
+    } catch (err) {
```

Comment:
@jeyem does this form of passing parameters resolve your concerns?

Reply:
why not some thing like `email(body.type, userId, message, identityName, addKwargs)` ?

Reply:
It can be yes, i thought that we might add some other options later on
I will change it to match your preferences

Reply:
@jeyem you mean that the last item would better not to be an object?
like
`
await email(body.type, userId, message, id, identityName, body.additionalKwargs?? {})
`
or more concise:
`
// will give default value of {} to the last parameter in "email" function
await email(body.type, userId, message, id, identityName, body.additionalKwargs)
`